docker/Dockerfile-netty-centos6

```diff
@@ -0,0 +1,22 @@
+FROM centos:6
+MAINTAINER netty@googlegroups.com
```

Comment:
I believe Google groups need sign up, right? If that's the case, this should be an email that a user of the image can contact without having to sign up for Google groups - all they want to do is report issues.

Reply:
no you can send an email to it even without sign up.